### PR TITLE
Added an option to start maximized

### PR DESCRIPTION
**Pull request by acidrain42**

### CODE REVIEW

**Comment by danbruegge** [comment]
Why isn't that merged? I think it is such a useful feature.

**Comment by thestinger** [comment]
Is there a use case for the configuration option? I can understand wanting the command-line switch, but not the other part.

**Comment by acidrain42** [comment]
I added the option because I want termite to always start maximized. In my case, I don't need the command-line switch but I added it for consistency and because I figured other people might find it useful.

**Comment by jelly** [comment]
Not sure if @thestinger  wants this still merged, but you are missing atleast the man page changes.

**Comment by acidrain42** [comment]
@jelly Good point, I updated the man pages just in case.

**Comment by Marcel-Robitaille** [comment]
Has there been any progress on this? I love termite and use it every day. I would really like this feature.

File: config [hunk 1]
```diff
@@ -5,6 +5,7 @@
 #dynamic_title = true
 font = Monospace 9
 #fullscreen = true
+maximized = false
```
Comment (polyzen) [inline config]: Please comment this out as it's the default anyhow.

Comment (acidrain42) [inline config]: Done

**Comment by Iambecomeroot** [comment]
Any idea when well see this get merged?

**Comment by jelly** [comment]
I agree with @thestinger I think it's sensible as a command line option, since that can be easily bound to a keybinding. Otherwise re-open it and we can continue the diiscussion.

**Comment by Iambecomeroot** [comment]
Seems reasonable. I use a tiling wm now so I don't care anymore.

**Comment by b-fg** [comment]
I would like this feature as well, even if only as a command line option.

**Pull request closed without merging**